rclcpp_action/src/client.cpp

```diff
@@ -353,16 +353,6 @@ ClientBase::is_ready(const rcl_wait_set_t & wait_set)
 
   pimpl_->next_ready_event = std::numeric_limits<size_t>::max();
 
-  if (is_feedback_ready) {
```

Comment:
can you add here as a comment why the order matters (i.e. what you wrote in the PR description)

Reply:
done in e66d6e514dadbc704c49c268078483ecc03a305a